wp-content/themes/humanity-theme/includes/theme-setup/scripts-and-styles.php

```diff
@@ -347,3 +347,26 @@ function amnesty_disable_cart_fragments() {
 }
 
 add_action( 'wp_enqueue_scripts', 'amnesty_disable_cart_fragments', 200 );
+
+if ( ! function_exists( 'amnesty_force_thousands_separator_localisation' ) ) {
+	/**
+	 * Localise the thousands separator option
+	 *
+	 * @package Amnesty\ThemeSetup
+	 *
+	 * @return void
+	 */
+	function amnesty_force_thousands_separator_localisation() {
+		$force_thousands_separator = get_option( 'amnesty_localisation_options_page' )['force_thousands_separator'] ?? 'off';
+
+		$data = [
+			'forceTS' => $force_thousands_separator,
```

Comment:
If you cast to boolean here, the script can simply check the truthiness of the value

```suggestion
			'forceTS' => amnesty_validate_boolish( $force_thousands_separator ),
```

Reply:
resolved